File: server/router/admin/gost_node_rule.go
package admin

import (
	"github.com/gin-gonic/gin"
	"server/controller/admin/gost_node_rule"
	"server/global"
	"server/router/middleware"
)

func InitGostNodeRule(group *gin.RouterGroup) {
	g := group.Group("gost/node/rule", middleware.Auth(global.Jwt), middleware.AuthAdmin())
	g.POST("list", gost_node_rule.List)
}

File: web/src/api/admin/gost_node_rule.js
import request from "../../request/index.js";

const baseUrl = '/v1/admin/gost/node/rule'

export const apiAdminGostNodeRuleList = (data) => {
    return request.request({
        url: `${baseUrl}/list`,
        method: 'POST',
        data
    })
}
